oasislmf/computation/generate/files.py

```diff
@@ -232,14 +234,13 @@ def run(self):
             model_settings = ModelSettingSchema().get(self.model_settings_json)
             correlations = establish_correlations(model_settings=model_settings)
             try:
-                model_damage_group_fields = model_settings["data_settings"].get("damage_group_fields")
+                model_damage_group_fields = model_settings.get("data_settings").get("damage_group_fields")
             except (KeyError, AttributeError, OasisException) as e:
-                self.logger.warn('WARNING: Failed to load {} - {}'.format(self.model_settings_json, e))
-
+                self.logger.warn(f'WARNING: Failed to load "damage_group_fields", file: {self.model_settings_json}, error: {e}')
```

Comment:
Are we sure we want to proceed if data_settings are not there?
I think this is risky, and potentially hard to debug.
I'd access `data_settings` before the try..except block so that the code errors if data_settings is not there, or access `data_settings` in its own try..except to log that nicely and then pass through the exception anyway so that the code errors.

Reply:
Yes, that's a good spot, i'll revert the access to `data_settings` to what it was